locale/surveyForm/nn.yaml

```diff
@@ -0,0 +1,2 @@
+select:
+    default: Velge et alternativ
```

Comment:
Same problem here, and google translate confirms it:

<img width="559" alt="image" src="https://github.com/zetkin/call.zetk.in/assets/550212/74dfd4df-0243-4948-91d8-164995eb91cf">

Reply:
Alright! I changed it to vaelge and velge respectively.

Reply:
I see that you've changed it to _vælg_ and _velg_, which I think is correct!